tests/unit/test_file_checker.py

```diff
@@ -39,7 +39,6 @@ def test_raises_exception_on_failed_plugin(tmp_path, default_options):
         "parameters": dict(),
         "plugin": mock.MagicMock(side_effect=ValueError),
     }
-    """Verify a failing plugin results in an plugin error"""
```

Comment:
Frankly, I don't know why this `tox -e linters` error appeared. Has nothing to do with the PR itself.